apps/workers/searchWorker.ts

```diff
@@ -72,6 +73,13 @@ async function runIndex(
             content: bookmark.link.content,
           }
         : undefined),
+      ...(bookmark.asset
+        ? {
+            content: bookmark.asset.content,
+            info: bookmark.asset.info,
+            metadata: bookmark.asset.metadata,
+          }
+        : undefined),
```

Comment:
I'm a bit surprised that typescript didn't complain about the fields used here as they are not defined in the search schema in `packages/shared/search.ts`